packages/trip-details/src/fare-table.tsx

```diff
@@ -159,14 +122,49 @@ const FareTypeTable = ({
           <tr key={index}>
             <td className="no-zebra">{leg.routeShortName}</td>
             {colsToRender.map(col => {
-              const fare = leg.fares[col.key];
+              let fare;
+              if (!hasLegProducts) {
+                fare = leg.fares[col.key];
+              } else {
+                fare = getLegCost(leg, col.riderCategory, col.fareContainer);
+              }
               return (
-                <td key={col.key}>
+                <td
+                  key={col.key}
+                  title={
+                    fare.transferAmount &&
+                    intl.formatMessage(
+                      {
+                        defaultMessage:
+                          defaultMessages[
+                            "otpUi.TripDetails.transferDiscountExplanation"
+                          ],
+                        description:
+                          "Text explaining the transfer discount applied to this fare.",
+                        id: "otpUi.TripDetails.transferDiscountExplanation"
+                      },
+                      {
+                        transferAmount: intl.formatNumber(
+                          fare.transferAmount / 100,
+                          {
+                            currency: fare?.price?.currency?.currencyCode,
+                            currencyDisplay: "narrowSymbol",
+                            style: "currency"
+                          }
+                        )
+                      }
+                    )
+                  }
+                >
+                  {(fare?.isTransfer || fare.transferAmount) && (
+                    <>
+                      <TransferIcon size={16} />{" "}
```

Comment:
`paddingRight: '1ch'` instead of a space? Then can remove the fragment

Reply:
I think the space is preferred for LTR languages.